dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/parameters/AnalyticsJobParameters.java

```diff
@@ -55,15 +55,18 @@ public class AnalyticsJobParameters implements JobParameters {
   private Set<String> skipPrograms = new HashSet<>();
 
   @JsonProperty private boolean skipResourceTables = false;
+  @JsonProperty private boolean skipOutliers = false;
 
   public AnalyticsJobParameters(
       Integer lastYears,
       Set<AnalyticsTableType> skipTableTypes,
       Set<String> skipPrograms,
-      boolean skipResourceTables) {
+      boolean skipResourceTables,
+      boolean skipOutliers) {
```

Comment:
I see that the `skipOutliers` flag is delivered until the table export. However, I could not find where this is used to export or not the data (skip or not outliers data)...

Reply:
done